### ISSUE TITLE
Remove link to user's Github account from Profile Page (currently unintuitive)

### ISSUE THREAD

**Issue by Engineering-Robert** [vinitkumar/node-twitter#196]
### User story
What problem is the feauture trying to solve for the user

### Conditions of satisfaction
Define what must be true for the feature to be considered done

### Additonal specifications
Give some extra specifications if needed like screenshots and designs

### Steps to reproduce
1.
2.
3.

### Expected behaviour
Tell what should happen

### Actual behaviour
Tell what happens instead

### Client configuration

**Browser:**
**Operating system:**

#### Web server error log

```
Insert your webserver log here
```

**Comment by arshadkazmi42** [comment]
@robertcoopercode I would like to take this up. Is this requirement still available to grab?

**Comment by robertcoopercode** [comment]
@arshadkazmi42 Go for it buddy! Let me know if you get stuck at all and I'll help you out.

**Comment by arshadkazmi42** [comment]
For this, I just have to remove the hyperlink from username in profile page. Is that right?

**Comment by robertcoopercode** [comment]
That's right. Make sure to test it locally and provide a screenshot when making the PR please.

**Comment by arshadkazmi42** [comment]
I am almost done with that change. So it looks something like this now

![screen shot 2018-08-12 at 11 29 03 pm](https://user-images.githubusercontent.com/4654382/44004861-905fcdb4-9e87-11e8-8069-e04fc23d834e.jpg)

Does this looks fine? If its fine, then I can create a PR

**Comment by robertcoopercode** [comment]
Nice man! Ya, go ahead and make a PR 👍

**Comment by arshadkazmi42** [comment]
@robertcoopercode  PR created [218](https://github.com/vinitkumar/node-twitter/pull/218)
